src/mcts/search.h

```diff
@@ -236,13 +241,17 @@ class SearchWorker {
   // 7. Update the Search's status and progress information.
   void UpdateCounters();
 
+  struct Bounds {
```

Comment:
What does this structure do? I think it deserves a better name, as it's not clear what it is for.
Why is it public if the only function that uses it is private?

Reply:
It is a struct that contains the bounds of a position and if tb-hits are used in proving the bounds. Its mainly used as a return struct in the negabound search. Made it private. As for a better name, hmmm.... maybe ProvenBounds or SearchResultBounds...., i still like Bounds. Ideas?